number_theory/modular.h

```diff
@@ -134,20 +135,20 @@ namespace modular_internal {
 
 // Tests if the type T is the Modular class.
 template <typename T>
-struct IsModularImpl {
+struct is_modular {
```

Comment:
Why did we switch naming from MixedCase to snake_case here? I thought we were using MixedCase for class/struct naming. Does this struct follow a common std naming etc.?

Reply:
I was following the `is_meowable` example. Let's use MixedCase here.